### PR TITLE
doc: (micro change) changed (C) to ©

### PR DESCRIPTION
**Pull request by Shreyas-SAS**
Pull Request for Issue # .

### Summary of Changes

**ISSUE:**
previous README.md used improper copyright symbol rather than the officially accepted symbol. 

**Proposed CHANGES:**
Changed Copyright symbol to increase professionalism of repository README. 
Also enhances legitimacy in legal terms as previous symbol had no legal meaning.

### Testing Instructions

Not Applicable.

### Actual result BEFORE applying this Pull Request

Not Applicable.

### Expected result AFTER applying this Pull Request

Not Applicable.

### Documentation Changes Required

**Proposed CHANGES:**
Changed Copyright symbol to increase professionalism of repository README. 
Also enhances legitimacy in legal terms as previous symbol had no legal meaning.

### CODE REVIEW

**Comment by RickR2H** [comment]
I have tested this item :white_check_mark: successfully on 788719f25415f0afc131fad34827345248e5522e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37271">issues.joomla.org/tracker/joomla-cms/37271</a>.</sub>

**Comment by brianteeman** [comment]
> Also enhances legitimacy in legal terms as previous symbol had no legal meaning.

you do not have to use the copyright symbol to obtain copyright protection for it; such protection is automatic.

**Comment by Shreyas-SAS** [comment]
> you do not have to use the copyright symbol to obtain copyright protection for it; such protection is automatic. 

I absolutely agree with you sir. Maybe I could not present it properly the first time.

What I meant is that the symbol shows more legitimacy than (C). On the other hand (C) is not a recognized symbol legally while copyright symbol is. Although the copyright act will still be active its always better to use a proper symbol on the Main Page of the repository.

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 788719f25415f0afc131fad34827345248e5522e<br /><br />Nice :) <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37271">issues.joomla.org/tracker/joomla-cms/37271</a>.</sub>

**Comment by chmst** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37271">issues.joomla.org/tracker/joomla-cms/37271</a>.</sub>

**Comment by bembelimen** [comment]
If this should be merged, we still need the space after the copyright sign...but I'm yet not sure about it, as Joomla! uses (C) everywhere, so probably need a production decision to change it everywhere. (Pinging @nibra as if I remember correctly you handled the copyright somewhere in the past...)

**Comment by Shreyas-SAS** [comment]
> If this should be merged, we still need the space after the copyright sign...but I'm yet not sure about it, as Joomla! uses (C) everywhere, so probably need a production decision to change it everywhere. (Pinging @nibra as if I remember correctly you handled the copyright somewhere in the past...)

@bembelimen  @nibra @chmst 

Of what I have noticed in the repository, most of the copyright symbol are set to (C) form, but I only wish to propose a change of these in the Documentation for the repository only and not the codes where it is usually present in the top of files as a comment. 

The reason that the change was proposed only in the main Documentation file of the Repo (README.md) was due to its extensive visibility, Every person that visits the repo goes through the readme if not all documentation. In short its the face of the repository and I wanted to keep it as formal and clean as possible.

I would also like to add that I will be ready to take up the task of replacing each and every (C) in the repo with © after the date of 4th April since I am trying to get a project for GSOC currently and going through each file to find and replace (C) will not be feasible right now.

I am really thankful to you for your time and sorry if I brought you any trouble.

File: README.md [hunk 1]
```diff
@@ -74,6 +74,6 @@ Do you want to improve Joomla?
 
 Copyright
 ---------------------
-* (C) 2005 Open Source Matters, Inc. <https://www.joomla.org>
+* ©2005 Open Source Matters, Inc. <https://www.joomla.org>
```
Comment (richard67) [inline README.md:77]:
```suggestion
* © 2005 Open Source Matters, Inc. <https://www.joomla.org>
```
Still it needs to add back the space, or not?

**Comment by laoneo** [comment]
Thanks for your contribution. Unfortunately we would like to keep it consistent in the repo and stay with the (C) syntax. If you want to change to the symbol it would require an adaption in every file where we use the doc header. So I'm closing this pr but there are many more issues where we can use your help.

**Comment by Shreyas-SAS** [comment]
> Thanks for your contribution. Unfortunately we would like to keep it consistent in the repo and stay with the (C) syntax. If you want to change to the symbol it would require an adaption in every file where we use the doc header. So I'm closing this pr but there are many more issues where we can use your help.

I completely understand sir, I am really thankful for your time. I will try contributing to the community in every other way possible.

**Comment by nibra** [comment]
I agree that '(C)' should be displayed as '©' in the documention (i.e., *.md files), but the stress is on 'displayed'. When the document is used outside an editor or GitHub, like rendered on websites as documentation, '(C)' should be replaced with '©' on the fly by the rendering engine.

**Comment by Shreyas-SAS** [comment]
> I agree that '(C)' should be displayed as '©' in the documention (i.e., *.md files), but the stress is on 'displayed'. When the document is used outside an editor or GitHub, like rendered on websites as documentation, '(C)' should be replaced with '©' on the fly by the rendering engine.

Oh. I really didn't know that it will be rendered in the website to a proper symbol. 
I am thankful to you for haven taken out the time to respond to my silly commit. I have definitely learned a lot and will try my best to contribute something useful and feasible next time. 
Thank you once again.

**Pull request closed without merging**